### DISCUSSION TITLE
How can I delete an unused workspace?

### DISCUSSION THREAD

**Discussion by silverfantacy** [fief-dev/fief#288] [Q&A] [answered]
I want to connect to my own database, but all my attempts to create it have failed, resulting in many workspaces. However, I can't find a place to delete them.

I would also like to ask, how can I export or view the contents of the database if I use the official database for creating it?

**Comment by fief-bailiff** [comment]
Hail, @silverfantacy 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment, marked as answer]
Hi @silverfantacy 👋 

Could you send us email at contact@fief.dev with the email you used to register on Fief? I'll delete it for you.

As for exporting your data on Cloud database, you can either use the [Admin API](https://docs.fief.dev/api/) or if you prefer an SQL dump I could produce one for you 👍
